assets/model_monitoring/components/src/shared_utilities/store_url.py

```diff
@@ -91,7 +92,8 @@ def valid_aml_obo_credential():
                 aml_obo_credential = AzureMLOnBehalfOfCredential()
                 if validate_aml_obo_credential:
                     try:
-                        _ = aml_obo_credential.get_token()
+                        _ = aml_obo_credential.get_token("https://management.azure.com/.default")
+                        return aml_obo_credential
                     except CredentialUnavailableError as cue:
                         raise InvalidInputError(MISSING_OBO_CREDENTIAL_HELPFUL_ERROR_MESSAGE.format(message=str(cue)))
                 else:
```

Comment:
shouldn't you remove `else` so that it will return aml_obo_credential after validation check?